tools/rosbag/test/test_roundtrip.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+import rosbag
+import rospy
+
+class TestRoundTrip(unittest.TestCase):
+    def _write_simple_bag(self):
+        from std_msgs.msg import Int32, String
+
+        with rosbag.Bag('/tmp/test.bag', 'w') as bag:
```

Comment:
By default each nosetest invocation runs its test sequentially. Separate CTests (therefore separate nosetests) will run in parallel.

But since the user can override this with e.g. `NOSE_PROCESSES` I think it is good to avoid any cross talk if possible by design.